src/main/kotlin/bd/clinic/modules/patient/Patient.kt

```diff
@@ -10,6 +12,9 @@ class Patient(
         val id: Int? = null,
         val firstName: String,
         val lastName: String,
-        @OneToMany(mappedBy = "patient")
-        val examinations: MutableList<Examination>
+        @field:PESEL
+        val pesel: String,
+        @Embedded
+        val address: Address,
+        val creationDate: LocalDate = LocalDate.now()
```

Comment:
To bym może zmienił na `modifiedOn` albo `modificationDate` (?), bo to się nadpisuje przy każdej edycji encji